src/dispatchers.ts

```diff
@@ -463,6 +463,53 @@ export async function handleAnnoucedCreate(
     await addToList(ctx.data.db, ['inbox'], create.id.href);
 }
 
+export const createHandleUndo = (accountService: AccountService) =>
```

Comment:
Can we call this `createUndoHandler` to keep consistency with the other handlers that are created in this manner? i.e `createFollowHandler`